.travis.yml

```diff
@@ -1,22 +1,26 @@
 language: go
 
+dist: trusty
```

Comment:
Lock the build to trusty aka ubuntu 14.04.  Technically went EOL on 2019-04, but Travis supports it and 12.04 for the time being.